Tests/XcodeProjMapperTests/IntegrationTests/IntegrationTests.swift

```diff
@@ -0,0 +1,173 @@
+import Foundation
+import InlineSnapshotTesting
+import Path
+import Testing
+import XcodeProj
+import XcodeProjMapper
+@testable import XcodeGraph
+
+// The IntegrationTests suite is serialized and uses inline snapshot testing.
+// If a test fails, its snapshot can be re-recorded as needed by changing `record: .failed`.
+@Suite(
+    // TODO: - Remove this when no longer lazy unzipping fixtures
+    .serialized,
+    .snapshots(
+        record: .failed
+    )
+)
+struct IntegrationTests {
+    /// Asserts that the given graph parsed from a workspace fixture matches the provided inline snapshots.
+    ///
+    /// - Parameters:
+    ///   - fixture: A closure that returns a `WorkspaceFixture` used to generate the graph.
```

Comment:
Why are we asserting with snapshots? I'd instead compare the parsed `XcodeGraph` with the expected `XcodeGraph`:
```swift
#expect(gotXcodeGraph, expectedXcodeGraph)
```

What's the reason we went with asserting on snapshot text which is more unstable than comparing the `XcodeGraph` directly?